x-pack/test_serverless/functional/test_suites/observability/cypress/cypress.config.ts

```diff
@@ -30,4 +39,9 @@ export default defineCypressConfig({
     supportFile: './support/e2e.ts',
     specPattern: './e2e/**/*.cy.ts',
   },
+  env: {
+    username: kbnTestConfig.getUrlParts().username,
+    password: kbnTestConfig.getUrlParts().password,
+    kibanaUrl,
```

Comment:
@sqren and @achyutjhunjhunwala,
we are now getting the username and password from the package (Thanks @kpatticha for your help)
What do you think about passing them as environment variables to cypress?